inscriptions/ISic0249.xml

```diff
@@ -174,70 +179,20 @@
-                    <supplied reason="lost">anno</supplied>
-                    <lb break="no" n="3"/>
-                    <supplied reason="lost">rum plus</supplied>
-                    minus XLV
-                    <supplied reason="lost">die</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="4"/>
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>s</abbr>
-                            <ex>is</ex>
-                        </expan>
-                    </supplied>
-                    A
-                    <expan>
-                        <ex>u</ex>
-                        <abbr>g</abbr>
-                        <ex>usti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ind</abbr>
-                        <ex>ictione</ex>
-                    </expan>
-                    XV im
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>p</abbr>
-                            <ex>eratore</ex>
-                        </expan>
-                        <expan>
-                            <abbr>d</abbr>
-                            <ex>omino</ex>
-                        </expan>
-                        <expan>
-                            <abbr>n</abbr>
-                            <ex>ostro</ex>
-                        </expan>
-                    </supplied>
-                    <lb n="5"/>
-                    <supplied reason="lost">H</supplied>eraclio piissimo
-                    <lb n="6"/>
-                    <expan>
-                        <abbr>s</abbr>
-                        <ex>emper</ex>
-                    </expan>
-                    <expan>
-                        <abbr>A</abbr>
-                        <ex>ugusto</ex>
-                    </expan>
-                    .
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">hic requi</supplied>esc<supplied reason="lost">it in pace</supplied>
+                    <lb n="2"/><supplied reason="lost">bone memo</supplied>riae A<gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">anno</supplied>
+                    <lb break="no" n="3"/><supplied reason="lost">rum plus</supplied> minus <num value="45">XLV</num> <supplied reason="lost">die</supplied> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4"/><supplied reason="lost"><expan><abbr>s</abbr><ex>is</ex></expan></supplied> A<expan><ex>u</ex><abbr>g</abbr><ex>usti</ex></expan> <expan><abbr>ind</abbr><ex>ictione</ex></expan> <num value="15">XV</num> im<supplied reason="lost"><expan><abbr>p</abbr><ex>eratore</ex></expan> <expan><abbr>d</abbr><ex>omino</ex></expan> <expan><abbr>n</abbr><ex>ostro</ex></expan></supplied>
```

Comment:
Something odd going on here? Not sure what the initial `<expan>` covers ? s(is)?
the subsequent A(u)g(usti) should, i think be: `<expan><abbr>A</abbr><ex>u</ex><abbr>g</abbr><ex>usti</ex></expan>`?
and after that: `im<supplied reason="lost"><expan><abbr>p</abbr><ex>eratore</ex></expan>` is one of those complex cases where the initial 'im' needs to be within the `<expan>`, which means you'll need two separate `<supplied>` elements, one within the `<abbr>` and then one for the `<ex>`.

Reply:
Yes, we can go over these, as they are not super intuitive in the first instance.

Reply:
indeed, took me ages to get to grips with it!